config/import-control.xml

```diff
@@ -40,20 +43,13 @@
   <allow class="java.lang.annotation.Target" local-only="true"/>
   <allow class="com.puppycrawl.tools.checkstyle.xpath.XpathQueryGenerator" local-only="true"/>
 
-  <!-- allowed till https://github.com/checkstyle/checkstyle/issues/3455 -->
-  <allow class="com.google.common.base.CaseFormat" local-only="true"/>
-  <allow class="com.google.common.io.BaseEncoding" local-only="true"/>
-  <allow class="com.google.common.io.ByteStreams" local-only="true"/>
-
   <allow class="com.puppycrawl.tools.checkstyle.XpathFileGeneratorAstFilter"
          local-only="true"/>
   <allow class="com.puppycrawl.tools.checkstyle.XpathFileGeneratorAuditListener"
          local-only="true"/>
 
   <file name="PropertyCacheFile">
-    <!-- allowed till https://github.com/checkstyle/checkstyle/issues/3455 -->
-    <allow class="com.google.common.io.Closeables"/>
-    <allow class="com.google.common.io.Flushables"/>
+    <allow class="com.google.common.io.BaseEncoding"/>
```

Comment:
This is allowed as I am not finding an easy to use implementation of Base 16 and I am not sure how stable it would be with pitest.

Reply:
```
BaseEncoding.base16().upperCase().encode(byteArray);
```
can be replaced with
```
new BigInteger(1, byteArray).toString(16).toUpperCase();
```

Reply:
Done.

Reply:
@pbludov 
> `toUpperCase();`

Since we have to specify the locale, will this cause problems for foreign languages if we specify `Locale.ROOT`?

Reply:
yes, it should be `toUpperCase(Locale.ROOT);`